lib/image_optim/cmd.rb

```diff
@@ -44,6 +86,38 @@ def check_status!
 
         fail SignalException, status.termsig
       end
+
+      def init_options!(args)
+        pgroup_opt = Gem.win_platform? ? :new_pgroup : :pgroup
+
+        if args.last.is_a?(Hash)
+          args.last[pgroup_opt] = true
+        else
+          args.push(pgroup_opt => true)
+        end
+      end
+
+      def cleanup_process(pid)
+        Thread.new do
+          Process.kill('-TERM', pid)
```

Comment:
Hmm I want to be killing the process group here since we won't know if the command being run will spawn child processes. My understanding is that sending a signal to the process group would also send the signal to all the processes in the process group.